sdcm/gemini_thread.py

```diff
@@ -94,10 +94,10 @@ def __init__(self, test_cluster: BaseCluster | BaseScyllaCluster, oracle_cluster
             "min-partition-keys": 2,
             "max-clustering-keys": 4,
             "min-clustering-keys": 2,
-            "partition-key-distribution": "normal",  # Distribution for hitting the partition
+            "partition-key-distribution": "zipf",  # Distribution for hitting the partition
```

Comment:
Why distribution type has been changed?

Reply:
reverted to `normal`, but in testing, `zipf` has been a lot better then `normal`